wrapanapi/systems/rhevm.py

```diff
@@ -29,12 +29,18 @@ class _SharedMethodsMixin(object):
     def _identifying_attrs(self):
         return {'uuid': self._uuid}
 
-    def refresh(self):
+    def refresh(self, **kwargs):
         """
         Re-pull the data for this entity and update this instance's attributes
+
+        Args:
+            kwargs: passed to VmService or TemplateService get() call
+
+        http://ovirt.github.io/ovirt-engine-sdk/master/services.m.html   (next line for specifics)
+            #ovirtsdk4.services.TemplateService.get
```

Comment:
was this meant to be removed?

Reply:
I see you have a comment similar below, can ignore the comment about deleting.

Reply:
no, for future reference